src/OVAL/probes/SEAP/sch_pipe.c

```diff
@@ -31,7 +31,9 @@
 #include <limits.h>
 #include <sys/stat.h>
 #include <sys/uio.h>
+#if !defined(_AIX)
 #include <sys/unistd.h>
```

Comment:
are you sure this is not available on AIX?

Reply:
On my AIX, I've got this compiler error: 1506-296 (S) #include file <sys/unistd.h> not found.
But beyond AIX, are you sure that sys/uio.h, sys/unistd.h and even fcntl.h are really necessary?
On my CentOS, Solaris and AIX boxes, I can compile sch_pipe.c without these 3 includes.

Reply:
In that case just remove the include, it could be redundant/forgotten.